corehq/apps/users/static/users/js/roles.js

```diff
@@ -267,22 +279,6 @@ hqDefine('users/js/roles',[
                         allowCheckboxId: null,
                         allowCheckboxPermission: null,
                     },
-                    {
-                        showOption: root.webAppsPrivilege,
```

Comment:
I'm guessing this was for projects that have plans without access to web apps (is that Community?).  Removing this restriction will mean that those projects will now be able to configure web apps permissions even when not applicable.  Probably not a big deal, just wanted to check.

Reply:
nevermind, I see it was functionally replaced with
```            {% if request|request_has_privilege:"CLOUDCARE" %}```
below